examples/functions-as-entry-points/functions_as_entry_points.bal

```diff
@@ -0,0 +1,52 @@
+import ballerina/io;
+
+error unknownOpError = { message: "unknown operation" };
+
+// A function named `main` acts as a default entry point to a Ballerina program.
+// This entry function could have zero or more parameters and could either have no
+// returns or could return an int.
```

Comment:
Minor change:
"// This entry function can have zero or more parameters, and can either return an int or can have no
// return value."